guides/common/modules/services.adoc

```diff
@@ -0,0 +1,92 @@
+[[Services]]
+= Services
+
+{ProjectName} uses a set of back-end services to perform the actual job. The status of these services can negatively influence the whole system and it’s one of the first things to check when some errors occur.
```

Comment:
What job?

Reply:
I took this from the Foreman docs, I am open to a different wording to make it sound better.